.pylintrc

```diff
@@ -16,7 +12,8 @@
 # R0913 Too many arguments
 # R0904 Too many public methods
 # E0211: Method has no argument
-disable=C0103,C0111,I0011,I0012,W0704,W0142,W0212,W0232,W0613,W0702,R0201,W0614,R0914,R0912,R0915,R0913,R0904,R0801
+disable=C0103,C0111,I0011,W0212,W0613,W0702,R0201,W0614,R0914,R0912,R0915,R0913,R0904,R0801
+load-plugins=pylint.extensions.no_self_use
```

Comment:
The other pylintrc changes were made in #10, but not this line. What is this extension? Do newer pylint versions require it?